specifyweb/frontend/js_src/lib/components/AppResources/types.tsx

```diff
@@ -203,6 +203,20 @@ export const appResourceSubTypes = ensure<IR<AppResourceSubType>>()({
     icon: icons.viewList,
     label: resourcesText.otherPropertiesResource(),
   },
+  preferences: {
+    mimeType: 'text/x-java-properties',
+    name: 'Preferences',
+    documentationUrl: undefined,
+    icon: icons.viewList,
+    label: preferencesText.preferences(),
+  },
+  globalPreferences: {
+    mimeType: 'text/x-java-properties',
+    name: 'Global Preferences',
+    documentationUrl: undefined,
+    icon: icons.viewList,
+    label: resourcesText.globalPreferences(),
+  },
```

Comment:
we are not encouraging users to create these app resources (they should be created by sp6 in every database). In fact, we should deprecate these app resources as soon as Institutional Preferences are added), thus we should not be exposing explicit options for these legacy resources in the UI.